.github/workflows/daily.yml

```diff
@@ -937,5 +937,5 @@ jobs:
       with:
         path: "./utils/req-res-validator/requirements.txt"
     - name: validator
-      run: ./utils/req-res-log-validator.py --verbose --fail-missing-reply-schemas --fail-commands-not-all-hit
+      run: ./utils/req-res-log-validator.py --verbose --fail-missing-reply-schemas ${{ (!contains(github.event.inputs.skiptests, 'redis') && !contains(github.event.inputs.skiptests, 'module') && !contains(github.event.inputs.sentinel, 'redis') && !contains(github.event.inputs.skiptests, 'cluster')) && '--fail-commands-not-all-hit' || '' }}
```

Comment:
Let's also test that `test_args` is empty

Reply:
updated, i additionally modified the echo to print out the all the input better, let me know what you think